### PR TITLE
ola: update to 0.10.8

### PR DESCRIPTION
**Pull request by peternewman**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
None - I don't have an OS X test machine available anymore

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint`? - I don't have an OS X test machine available anymore
- [ ] tried existing tests with `sudo port test`? - I don't have an OS X test machine available anymore
- [ ] tried a full install with `sudo port -vst install`? - I don't have an OS X test machine available anymore
- [ ] tested basic functionality of all binary files? - I don't have an OS X test machine available anymore

### CODE REVIEW

**Comment by mf2k** [comment]
Error is 
`Error: No checksum set for ola-0.10.8.tar.gz`

Indeed you removed the checksums in your PR.

File: net/ola/Portfile [hunk 1]
```diff
@@ -22,23 +21,10 @@ long_description    The Open Lighting Architecture (OLA) provides a plugin \
 
 homepage            http://www.openlighting.org/ola/
 
-checksums           rmd160  1312a43f8db8f211f68cd1ab390eb9c8bf476144 \
-                    sha256  8a65242d95e0622a3553df498e0db323a13e99eeb1accc63a8a2ca8913ab31a0 \
-                    size    2512577
```
Comment (mf2k) [inline net/ola/Portfile:27]: The checksums line needs to be restored and updated for the new version.

File: net/ola/Portfile [hunk 2]
```diff
@@ -4,8 +4,7 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cxx11 1.1
 
-github.setup        OpenLightingProject ola 0.10.7
-revision            2
```
Comment (mf2k) [inline net/ola/Portfile:8]: Please restore the `revision` line and set it back to 0.

**Review by mf2k — CHANGES_REQUESTED** [review]

File: net/ola/Portfile (hunk 2 above)
Comment (peternewman) [reply net/ola/Portfile:8]: Sorry, I think the previous time I'd reset I just removed it.

File: net/ola/Portfile (hunk 1 above)
Comment (peternewman) [reply net/ola/Portfile:27]: Done, after I found how to generate your rmd160 hash on your docs.

**Comment by peternewman** [comment]
> Port ola **fail** on xcode7.3. [Log](https://paste.macports.org/efd6912148b2)

This is due to this typo, which I've just fixed:
https://github.com/OpenLightingProject/ola/pull/1687/commits/f907b4c992e85bfb30559b5855c41194d2c52050

I don't know if it's worth doing a patch just for this one tweak or waiting for a future release. Do many people still use xcode7.3?

**Review by mf2k — APPROVED** [review]

**Comment by ryandesign** [comment]
> * [x]  squashed and [minimized your commits](https://guide.macports.org/#project.github)?

Oops, I didn't look closely enough to see that this PR had not minimized its commits. In the future please either do minimize your commits or else clearly indicate that you did not so that we know that we need to use the squash function when we merge.

**Comment by peternewman** [comment]
> > * [x]   squashed and [minimized your commits](https://guide.macports.org/#project.github)?
> 
> Oops, I didn't look closely enough to see that this PR had not minimized its commits. In the future please either do minimize your commits or else clearly indicate that you did not so that we know that we need to use the squash function when we merge.

Sorry @ryandesign . FWIW It was accurate when I opened the PR/made the commit, it just all went a bit downhill after that!

If it's a bit issue, it looks like you might be able to force squash by disabling merge, see:
https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuring-commit-squashing-for-pull-requests

**Comment by ryandesign** [comment]
We don't want to disable the ability to rebase and merge a PR, because it is often appropriate for a PR to contain multiple logically distinct commits.

**Pull request merged**